### DISCUSSION TITLE
utility has don't working

### DISCUSSION THREAD

**Discussion by eymenhuda** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#225] [Q&A] [answered]
i am using ryzen 5 3600 and big sur. after installing and restarting, the bar says "no conncection". when i click on it, the utility does not open. could you help?

**Comment by chris1111** [comment]
what is the brand and model of Wifi USB?

**Reply by eymenhuda** [reply]
ralink rt3070 chip

**Comment by chris1111** [comment, marked as answer]
its a Mediatek drivers, read the post, Mediatek not support here

Here [Mediatek](https://github.com/chris1111/D-LinkUtility-Package)
